.pipelines/containers/container-template-windows.yaml

```diff
@@ -11,19 +15,29 @@ steps:
     command: 'login'
     addPipelineData: false
 
-- powershell: |
-    echo "using tag ${{ parameters.tag }}"
-    . .\build\scripts\windows.ps1; ${{ parameters.name }}-image windows-${{ parameters.arch }}-${{ parameters.tag }} 
-  name: image_build
+- task: Docker@2
   displayName: Image Build
-  retryCountOnTaskFailure: 3  
+  retryCountOnTaskFailure: 3
+  inputs:
+    command: 'build'
+    containerRegistry: $(ACR_SERVICE_CONNECTION)
+    repository: 'azure-${{ parameters.name }}'
+    tags: 'windows-${{ parameters.arch }}-${{ parameters.tag }}'
+    Dockerfile: '${{ parameters.name }}/windows.Dockerfile'
+    arguments: '
+    --build-arg VERSION=windows-${{ parameters.arch }}-${{ parameters.tag }} 
```

Comment:
it looks like --build-arg is causing some errors in npm conformance test windows image builds

https://msazure.visualstudio.com/One/_build/results?buildId=63255597&view=logs&j=d229707a-47a3-50aa-1e61-a9a0b4c37ede&t=332ad56f-85e0-5dad-680f-4202a0f5b9d7&l=102

Reply:
Thanks, fixed this issue